apps/admin-portal/src/components/server-configurations/user-self-registration.tsx

```diff
@@ -268,127 +278,208 @@ export const UserSelfRegistration: FunctionComponent<UserSelfRegistrationProps>
 
 	const getFormValues = (values) => {
 		return {
+			accountLockOnCreation: selfSignUpConfigs.accountLockOnCreation,
 			callbackRegex: values.get(ServerConfigurationsConstants.CALLBACK_REGEX),
-			checkboxValues: values.get("SelfRegistrationCheckBoxes"),
+			enable: selfSignUpConfigs.enable,
+			internalNotificationManagement: selfSignUpConfigs.internalNotificationManagement,
+			reCaptcha: selfSignUpConfigs.reCaptcha,
 			smsOTPExpiryTime: values.get(ServerConfigurationsConstants.SMS_OTP_EXPIRY_TIME),
 			verificationCodeExpiryTime: values.get(ServerConfigurationsConstants.VERIFICATION_CODE_EXPIRY_TIME)
 		}
 	};
 
-	const showUserSelfRegistrationView = editingForm[USER_SELF_REGISTRATION_FORM_IDENTIFIER] ? (
+	const userSelfRegistrationSummary = (
 		<EditSection>
-			<Forms
-				onSubmit={ (values) => {
-					setSelfSignUpConfigs(getFormValues(values));
-					handleSubmit();
-				} }
-				resetState={ reset }
-			>
+			<Forms>
 				<Field
-					name="SelfRegistrationCheckBoxes"
+					name={ ServerConfigurationsConstants.SELF_REGISTRATION_ENABLE }
 					required={ false }
 					requiredErrorMessage=""
 					type="checkbox"
 					children={ [
 						{
-							label: t("devPortal:components.serverConfigs.selfRegistration.form.enable.label"),
+							label: t("devPortal:components.serverConfigs.selfRegistration.form." +
+								"enable.label"),
 							value: ServerConfigurationsConstants.SELF_REGISTRATION_ENABLE
-						},
+						}
+					] }
+					value={ selfSignUpConfigs.enable }
+					listen={
+						(values) => {
+							const value = values.get(ServerConfigurationsConstants.SELF_REGISTRATION_ENABLE).length > 0
```

Comment:
Does this line exceeds 120 character limit? Is so shall we break it into two lines?

Reply:
No. it's at the right length.